### ISSUE TITLE
Chunk glyph data, instead of scattering across tables.

### ISSUE THREAD

**Issue by hodefoting** [be-fonts/better-ergonomics-spec#1]
For networked (or otherwise split) renderers it is useful to be able to progressively transmit the glyphs of a font between front-end and backend. Both for normal use or overriding/specifying individual glyphs. This is hampered by the set of long tables design which scatters random access all around the font file. Departing from the set of tables and instead storing the meta-data relevant to an outline together with the shape more easily permits both streaming subsetting and better handling of large fonts on RAM constrained devices like microcontrollers. In https://ctx.graphics/ 's internal font representation I am already doing this for kerning data which follows the path vertices - and if this format gets expanded; I will add more commands for key/value pairs and similar along with this kerning information.

I've put this under ergonomics - since it is better ergonomics for the caches of the cpu, as well as well as potentially being better suited for some authoring format purposes.

**Comment by davelab6** [comment]
I agree, I think a key way to evaluate the ergonomics of a new format will be how well it plays with https://github.com/w3c/PFE/ :)

**Comment by simoncozens** [comment]
See also https://github.com/simoncozens/sff-spec
